internal/database/assigned_owners.go

```diff
@@ -102,6 +104,17 @@ func (s assignedOwnersStore) DeleteOwner(ctx context.Context, assignedOwnerID in
 	return err
 }
 
+const countAssignedOwnersFmtstr = `SELECT COUNT(*) FROM assigned_owners`
+
+func (s assignedOwnersStore) CountAssignedOwners(ctx context.Context) (int32, error) {
+	r := s.QueryRow(ctx, sqlf.Sprintf(countAssignedOwnersFmtstr))
+	var count int32
+	if err := r.Scan(&count); err != nil {
+		return 0, err
+	}
+	return count, nil
```

Comment:
nit: we can use `basestore`
```suggestion
	count, _, err := basestore.ScanFirstInt(s.db.Query(ctx, sqlf.Sprintf(countAssignedOwnersFmtstr)))
	if err != nil {
		return 0, err
	}
	return int32(count), err
```